First iteration of sase series (#20255)
cloudflare/cloudflare-docs: src/content/docs/learning-paths/sase-overview-course/series/video-3.mdx

---
pcx_content_type: navigation
title: Secure remote access to your critical infrastructure
sidebar:
  order: 4
prev: true
next: true
tableOfContents: false
---

import { Render, Tabs, TabItem, Stream, Card } from "~/components";

<Tabs>
  <TabItem label="Watch this episode">
  

>     In this video learn how Cloudflare's SASE platform can provide highly secure access to your critical infrastructure by leveraging a modern ZTNA service to implement zero trust principles. Applications, databases and their servers are running in a variety of locations. From on-premises data centers to cloud hyperscalers, making the need to secure administrative access more important than ever. 

Reviewer: ```suggestion
    In this video learn how Cloudflare's SASE platform can provide highly secure access to your critical infrastructure by leveraging a modern ZTNA service to implement Zero Trust principles. Applications, databases and their servers are running in a variety of locations from on-premises data centers to cloud hyperscalers, making the need to secure administrative access more important than ever. 
```

    <Card>
        <Stream
        videoId="a02e5f9e58999d96c3ec9dbb0efb9707"
        videoTitle="video 3"
        thumbnailTimeOrURL="https://pub-d9bf66e086fb4b639107aa52105b49dd.r2.dev/Simon-SASE-thumbnail-test.png"
        moreVideosLink="false"
      />

      **Related content**

>       If you want to dive into detail about using Cloudflare to access critical infrastructure, read here:

Reviewer 2: ```suggestion
      If you want to dive into detail about using Cloudflare to access critical infrastructure, refer to the following pages:
```

        - [Evolving to a SASE architecture with Cloudflare](/reference-architecture/architectures/sase/)
        - [SSH with Access for Infrastructure](/cloudflare-one/connections/connect-networks/use-cases/ssh/ssh-infrastructure-access/)

>         - [Using a zero trust framework to secure SaaS applications](/reference-architecture/design-guides/zero-trust-for-saas/)

Reviewer: ```suggestion
        - [Using a Zero Trust framework to secure SaaS applications](/reference-architecture/design-guides/zero-trust-for-saas/)
```

>         - [Building zero trust architecture into your startup](/reference-architecture/design-guides/zero-trust-for-startups/)

Reviewer: ```suggestion
        - [Building Zero Trust architecture into your startup](/reference-architecture/design-guides/zero-trust-for-startups/)
```
